New Validation Correlation rules - Missing search window (#3567)
demisto/demisto-sdk: CHANGELOG.md

* Fixed an issue where the (`GR108`) validation did not fail in the validate command with the `-a` flag.
* Modified **prepare_content** command to be platform specific. For xsoar-saas and XSIAM regarding pack readme and integration description images in markdown files.
* Fixed an issue where the **lint** command was parsing % that may exist in the log data.

> * Added a **validate** check in correlation rule files making sure that if `execution_mode` is set to `SCHEDULED`, `search_window` cannot be empty.

Reviewer: ```suggestion
* Added a **validate** check for correlation rule files, making sure that `search_window` cannot be empty when `execution_mode` is set to `SCHEDULED`.
```
